inhouse_bot/cogs/stats_cog.py

```diff
@@ -293,3 +212,74 @@ async def mmr_history(self, ctx: commands.Context):
             temp.close()
 
     # TODO MEDIUM PRIO (simple) Add !champions_stats once again!!!
+
+    @commands.command(aliases=["scout", "profile"])
+    @guild_only()
+    @doc(
+        f"""
+        Displays information about a player and champs used in inhouses
+        Example:
+            {PREFIX}scout @User
+            {PREFIX}profile @User
+    """
+    )
+    async def scouting(self, ctx: commands.Context, user: discord.Member):
+        with session_scope() as session:
+            player = (
+                session.query(Player)
+                .select_from(Player)
+                .filter(Player.server_id == ctx.guild.id)
+                .filter(Player.id == user.id)
+            ).one_or_none()
+
+            if not player:
+                await ctx.send("This player has no games played yet.")
+                return
+
+            # #### Create the initial embed object ####
+
+            # All Users default to unverified until further data is processed
+            embed = discord.Embed()
+            file = discord.File("riot-ranks/UNVERIFIED.png")
+            embed.set_thumbnail(url="attachment://UNVERIFIED.png")
+
+            if player.summoner_puuid:
+                summoner = await lol.get_summoner_by_puuid(player.summoner_puuid)
+                playerRankInfo = await lol.get_summoner_rank_info_by_id(summoner.id)
+                url = "https://www.op.gg/summoners/na/" f"{summoner.name}"
+                url = url.replace(" ", "")
+
+                # If a player has a rank whether it being (Solo/Duo or Flex or TFT)
+                if playerRankInfo:
+                    embed.set_author(
+                        name=summoner.name + f": {playerRankInfo['tier']}", url=url
+                    )
+                    file = discord.File(
+                        "riot-ranks/" + f"{playerRankInfo['tier']}" + ".png",
+                        filename=f"{playerRankInfo['tier']}" + ".png",
+                    )
+                    embed.set_thumbnail(
+                        url="attachment://" + f"{playerRankInfo['tier']}" + ".png"
+                    )
+
+                else:
+                    embed.set_author(name=summoner.name + ": UNRANKED", url=url)
+                    file = discord.File(
+                        "riot-ranks/UNRANKED.png", filename="UNRANKED.png"
+                    )
+                    embed.set_thumbnail(url="attachment://UNRANKED.png")
+
+            player_stats = await get_player_stats(ctx)
+            embed.add_field(
+                name="Role Stats", value="\n".join(player_stats), inline=False
```

Comment:
If the player_stats is empty then the embed will fail

Reply:
for this function I just moved over the !stats command logic, in the previous case if you do !stats with no ranked games it just displays an empty box of info so thats what would happen here

Reply:
There's an error when you add an empty field to an embed.
```
ERROR:inhouse_bot:400 Bad Request (error code: 50035): Invalid Form Body
In embeds.0.fields.0.value: This field is required
```

Reply:
I believe if you supply it an empty list it still displays

Reply:
Try it by hard coding the player_stats to []

Reply:
yea I just tried hmm I see, in his previous code he was just using the embed constructor, I can add something in get player stats to if the rating object is empty just return "no data found"